RELEASE_NOTES.md

```
@@ -1,3 +1,17 @@
+# Release Notes
+
+## v1.2.7
+
+### BREAKING CHANGES
+- Added `Len() int` method to `contracts.Cookies` interface
+
+### Features
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)